contrib/terraform/equinix/provider.tf

```diff
@@ -1,5 +1,11 @@
 terraform {
   required_version = ">= 1.0.0"
+
+  provider_meta "equinix" {
+    # Set the name of the module below 
+    module_name = "equinix-kubspray"
```

Comment:
```suggestion
    module_name = "equinix-kubespray"
```

Reply:
I think `kubespray` could be sufficient in this case, but the chosen name provides the needed context.